src/components/AssetAccountDetails.tsx

```diff
@@ -22,6 +24,7 @@ type AssetDetailsProps = {
 }
 
 export const AssetAccountDetails = ({ assetId: caip19, accountId }: AssetDetailsProps) => {
+  const marketData = useAppSelector(state => selectMarketDataById(state, caip19))
```

Comment:
Can we correct the return type on `selectMarketDataById` to `MarketData | undefined`?

Reply:
yes, we should but this breaks everything at the moment. the type def is actually wrong - your suggestion is correct.

root cause - enforce explicit return types in `tsconfig.json`